### PR TITLE
chore: Move "MDN at 10" under MDN folder

### PR DESCRIPTION
**Pull request by nschonni**
Ran `yarn content move MDN_at_ten MDN/At_ten`

### CODE REVIEW

File: files/en-us/mdn/at_ten/contributing_to_mdn/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Contributing to MDN
-slug: MDN_at_ten/Contributing_to_MDN
+slug: MDN/At_ten/Contributing_to_MDN
```
Comment (nschonni) [inline files/en-us/mdn/at_ten/contributing_to_mdn/index.html:3]: Not sure if this should be migrated/redirected to something under "MDN/Contributing"

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Looking mostly good, thanks @nschonni . I would like to see:

1. the contributing page redirected to MDN/Contributing. It is totally out of date and not very helpful.
2. The "Subnav" removed from the main "at ten" page. It is not serving any purpose, and the links are wrong now ;-)

Also, not sure if you want to fix the flaws in these documents now, or do that as a separate PR? I'm happy either way.

**Comment by nschonni** [comment]
@chrisdavidmills redirected the contributing. Left the subnav, but dropped the contributing link. Ran the flaws to fix the links and download some images

@peterbe I noticed the flaws actually downloaded the GitHub avatars, but I undid that since it didn't seem intentional to snapshot those

**Review by chrisdavidmills — APPROVED** [review]
LTGM; thanks @nschonni !

**Pull request merged**